### ISSUE TITLE
HTTPS Output (SNI)

### ISSUE THREAD

**Issue by runningman84** [buger/gor#389]
It would be great if gor would support a https endpoint.

Right now we get this error if the https endpoint is a cloudfront endpoint:
```
./gor --input-raw :8080 --output-http https://www.example.com --output-http-header "Host: www.example.com"

 [HTTPClient] Connection error: remote error: handshake failure
```
The same call works if we point it to the elastic loadbalancer...

The problem might be related to SNI?

**Comment by buger** [comment]
Well, it is definitely a bug, but I'm not sure yet what it can be. SSL code is quite simple, and rely on Go default TSL functionality: https://github.com/buger/gor/blob/master/http_client.go#L91-L99

**Comment by buger** [comment]
Closed with #390 thanks to @sattvik!

Feel free to use these binaries: 
[gor_fix_sni_x64.tar.gz](https://github.com/buger/gor/files/611053/gor_fix_sni_x64.tar.gz)
[gor_fix_sni_mac.tar.gz](https://github.com/buger/gor/files/611052/gor_fix_sni_mac.tar.gz)
